### ISSUE TITLE
one small but important bug

### ISSUE THREAD

**Issue by jind11** [nlpdata/mrc_bert_baseline#2]
hi, in the file of "run_classifier.py", you lost an code of "model.eval()" before you do the evaluation, e.g. around lines 539 and 540. In my experiments, I found this may result in a degradation of around 0.5% accuracy.

**Comment by accreator** [comment]
Could you clarify the issue a bit more? 
Given the current description, I checked the code but did not see the bug since line 575 is "model.eval()", which is before the actual evaluation. I also ran the code again but did not observe the performance difference no matter whether or not inserting "model.eval()" between line 539 and line 540.

**Comment by jind11** [comment]
Hi, I checked the code again, and yes, there is "model.eval()" already. I think it is my minor changes to the original that caused the discrepancy. Your code is right. Sorry for the bothering and thanks for the patience!
